scripted/sbt/src/main/scala/sbt/test/ScriptedTests.scala

```diff
@@ -399,17 +402,44 @@ class ScriptedRunner {
     reportErrors(tests.flatMap(test => test.apply().flatten.toSeq).toList)
   }
 
+  @deprecated("No longer used", "1.0.2")
```

Comment:
Let's make the version in deprecated here (and through out) 1.1.0.

Reply:
👍